embeddings/task/sklearn_task/text_classification.py

```diff
@@ -0,0 +1,43 @@
+from typing import Any, Dict, Optional, Union
+
+from numpy import typing as nptyping
+import pandas as pd
+from sklearn.base import ClassifierMixin as AnySklearnClassifier
+
+from embeddings.task.sklearn_task.sklearn_task import SklearnTask
+from embeddings.embedding.sklearn_embedding import SklearnEmbedding
+
+
+class TextClassification(SklearnTask):
+    def __init__(
+        self,
+        classifier: AnySklearnClassifier,
+        train_model_kwargs: Optional[Dict[str, Any]] = None,
```

Comment:
I think that naming should be corrected we used `train_model_kwargs` as args which are passed to `fit` I would use `classifier_kwargs` instead

Reply:
Done